### PR TITLE
Generate script to patch receipt chain hashes and last VRF output

### PR DESCRIPTION
**Pull request by psteckler**
Generate a script to patch the receipt chain hashes in the `zkapp_account_precondition` table, and the last VRF output in the `blocks` table.

There are three pieces here:

1) An app `receipt_chain_hash_to_b58`, a Unix-style filter that takes an `Fp` string as input and prints it as `Receipt.Chain_hash.t` in Base58Check format

2) A similar app, `last_vrf_output_to_b64`, that takes a hex string as input, and prints it as a Base64 string

2) A shell script that uses (1) and (2) to generate an SQL file that updates the archive db with the Base58Check hashes and Base64 string

These were tested against a local copy of the ITN archive db. After applying it, the replayer did not encounter the failures that were seen in the ITN replayer cron job.

Important:
- the script can be run as many times as needed; already-patched hashes and last VRF output are left intact
- make a backup of the archive data before running the SQL!

Part of #14417.

### CODE REVIEW

File: scripts/migrate-receipt-chain-hashes.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+if [ ! $# -eq 1 ] ; then
+    echo "Usage" $0 archive-db
+    exit 0
+fi
+
+ARCHIVE_DB=$1
+HASHES_FILE=hashes_file.tmp
```
Comment (dkijania) [inline scripts/migrate-receipt-chain-hashes.sh:9]: Shouldn't those be  a parameters ?

Comment (psteckler) [reply scripts/migrate-receipt-chain-hashes.sh]: Made them optional command-line args.

**Comment by psteckler** [comment]
!ci-build-me

**Review by dkijania — APPROVED** [review]

**Review by georgeee — APPROVED** [review]

**Comment by psteckler** [comment]
!ci-build-me

**Comment by psteckler** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!approved-for-mainnet

**Pull request merged**